gradio/blocks.py

```diff
@@ -685,16 +696,84 @@ async def call_function(self, fn_index, processed_input, iterator=None):
             "iterator": iterator,
         }
 
-    def postprocess_data(self, fn_index, predictions, state):
+    async def call_batch_function(
```

Comment:
Do we need a separate `call_batch_function` method?

Provided `processed_input` is the processed input for the batch isn't the following true?
```
call_batch_function(fn_index=0, processed_input_batch=batch) == call_function(fn_index, processed_input=batch, iterator=None)
```

Reply:
The main difference is that we pass in the input data as a list in `call_batch_function`: `block_fn.fn(processed_input_batch)` and we unpack the list in `call_function`: `block_fn.fn(*processed_input)`. 

I suppose this could be consolidated, though I'll wait until we finalize the API for batched functions

Reply:
Removed